packages/react-core/src/components/TextInputGroup/TextInputGroupMain.tsx

```diff
@@ -42,6 +44,7 @@ export const TextInputGroupMain: React.FunctionComponent<TextInputGroupMainProps
   className,
   icon,
   type = 'text',
+  hint: inputHint,
```

Comment:
Curious why this is needed?

Reply:
Probably not needed. I had an issue where it wasn't recognizing the new `hint` property so I was trying a bunch of different things. The `inputHint` format was me following the way they had `inputPlaceholder` and `inputValue`. The real issue ended up being that my IDE was acting up. Would you like me to remove this?